habit/src/containers/login/LoginScreen.js

```diff
@@ -2,18 +2,44 @@ import React from 'react';
 import { View } from 'react-native'
 import CustomTextInput from '../../components/TextInput'
 import * as CONSTANTS from '../../utils/string_constants'
+import {Button} from 'react-native-elements'
+import styles from '../../styles/login'
 
-import styles from '../../styles/common'
+export default class Login extends React.Component{
```

Comment:
Can we make this a functional component with hooks ?

Reply:
yes